### PR TITLE
Adjust wradlib provision

### PR DESCRIPTION
**Pull request by heistermann**
- wradlib is now hosted on GitHub
- course notebooks are part of a regular wradlib distribution
- example data is hosted on a different repo
- wradlib examples work with an environment variable for the sample data

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -40,15 +40,15 @@ Vagrant.configure("2") do |config|
   
   # Python environment, system
   config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_scipy_stack.sh"
-  config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_pyshp.sh"
+  #config.vm.provision :shell, :privileged => false, :path => "provision_scripts/install_pyshp.sh"
```
Comment (jjhelmus) [inline Vagrantfile]: Why not just remove these lines and the corresponding script if they are not needed.  We can always resurrect them from version control if they are needed.

File: provision_scripts/install_wradlib.sh [hunk 2]
```diff
@@ -7,17 +7,27 @@ set -x
...
-wget --no-check-certificate http://bitbucket.org/wradlib/wradlib/get/default.zip
-unzip default.zip
-cd wradlib-wradlib*
-sudo pip install -r requirements.txt
+# Install wradlib
+wget --no-check-certificate https://github.com/wradlib/wradlib/archive/master.zip
+unzip master.zip
+cd wradlib-master
+#sudo pip install -r requirements.txt
 sudo python setup.py install
```
Comment (jjhelmus) [inline provision_scripts/install_wradlib.sh]: Do any additional wradlib dependencies need to be installed?

**Comment by heistermann** [comment]
Sure, your're right - has been taken care of.

**Comment by jjhelmus** [comment]
> Sure, your're right - has been taken care of.

Do you want to remove the scripts also to keep the repo clean?

**Comment by heistermann** [comment]
Thanks for being strict - I promise to become more careful! Scripts removed.

**Comment by jjhelmus** [comment]
No problem :smile:.  Looks good to me

**Pull request merged**